src/styles/_transition.scss

```
@@ -62,3 +62,51 @@ $bk: 'background-shadow';
 .#{$bk}-leave-to {
   background-color: rgba(0, 0, 0, 0);
 }
+
+$tu: 'typing-users';
+.#{$tu}-enter-active {
+  transition: all 0.3s $easeOutCirc;
+}
+.#{$tu}-leave-active {
+  transition: all 0.3s $easeInCirc;
+}
```

消える時のtransition、ease outの方がいいかなと思ったんですがどうでしょうか？
あともうすこしゆっくり(0.5sぐらい)でもいいかな？と
```suggestion
.#{$tu}-enter-active {
  transition: all 0.5s $easeOutCirc;
}
.#{$tu}-leave-active {
  transition: all 0.5s $easeOutCubic;
}
```

---

確かにってなりました 😀